invenio_rest/config.py

```diff
@@ -22,7 +22,13 @@
 # waive the privileges and immunities granted to it by virtue of its status
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 
-"""Invenio REST configuration."""
+"""Invenio REST configuration.
+
+Please also see
+`Flask-Limiter <https://flask-limiter.readthedocs.io/en/stable/>`_ and
+`Flask-CORS <https://flask-cors.readthedocs.io/en/latest/>`_ for many more
```

Comment:
`Flask-CORS <https://flask-cors.readthedocs.io/en/latest/>`__

I think two underscores will solve the issue :)